### PR TITLE
Rename `get_{sender,receiver}` to `new_{sender,receiver}`

### PR DESCRIPTION
**Pull request by leandro-lucarella-frequenz**
Using `get_` as a prefix gives the idea that one is always getting the same object, but these methods are indeed creating new objects.

### CODE REVIEW

**Comment by leandro-lucarella-frequenz** [comment]
Rebased, ping @sahas-subramanian-frequenz

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ time.
 * You need to make sure to use [timezone-aware] `datetime` objects when using
   the timestamp returned by [`Timer`], Otherwise you will get an exception.
 
+* Channels methods `get_receiver()` and `get_sender()` have been renamed to
+  `new_receiver()` and `new_sender()` respectively. This is to make it more
+  clear that new objects are being created.
+
```
Comment (shsms) [inline RELEASE_NOTES.md:20]: Should this be listed under `Breaking changes`?

**Review by shsms — DISMISSED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (sahas-subramanian-frequenz) [inline RELEASE_NOTES.md:20]: Should this be listed under `Breaking changes`?

**Review by sahas-subramanian-frequenz — DISMISSED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (leandro-lucarella-frequenz) [reply RELEASE_NOTES.md:20]: Added a ` (breaking changes)` to `Upgrading` to make it more clear that this section list breaking changes. We can change the template if `Upgrading` is not clear enough. I used this to make it clear for people writing notes that it is important to say how to upgrade from breaking changes, not just listing them.

**Review by sahas-subramanian-frequenz — APPROVED** [review]

**Pull request merged**